lib/Elastica/Index/Settings.php

```diff
@@ -77,30 +77,32 @@ public function get($setting = '')
         }
         $settings = $data['settings']['index'];
 
-        if (!$setting) {
-            // return all array
+        /**
```

Comment:
If there is duplicated code it should be moved into a Util class and be used from both places.